docs/DeveloperGuide.md

```diff
@@ -365,22 +363,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
+* 2a. The list is empty.
+
```

Comment:
I think this extension 2a isn't supposed to be here anymore, its included in UC05 already.